packages/@react-spectrum/tooltip/docs/Tooltip.mdx

```diff
@@ -0,0 +1,175 @@
+```tsx example
+<TooltipTrigger>
+  <ActionButton aria-label="Edit Name"><Edit /></ActionButton>
+  <Tooltip>Change Name</Tooltip>
+</TooltipTrigger>
+```
+
+### Delay
+The above mentioned delay after hover can be adjusted. Focus will always appear instantly. Setting it to 0 will cause hover to be instant as well.
+```tsx example
+<TooltipTrigger delay={0}>
+  <ActionButton aria-label="Save"><Save /></ActionButton>
+  <Tooltip>Saving applies your new settings right away.</Tooltip>
+</TooltipTrigger>
+```
+
+## Content
+
+Tooltips can have a label provided as a child. This can be an internationalized string.
+
+### Accessibility
+
+Tooltips will use their content to describe their trigger, they should not be complex. Tooltip triggers must be focusable and hoverable in order to ensure that all users can activate them.
+
+## Tooltip placement
+
+Tooltips support a variety of placement options since they do not take over the user interface.
+
+### Placement
+
+The popover's placement with respect to its anchor element can be adjusted using the `placement`
+prop. See the props table [below](#tooltiptrigger-props) for a full list of available placement combinations.
+
+```tsx example
+<TooltipTrigger placement="end">
+  <ActionButton aria-label="Foo">Trigger</ActionButton>
+  <Tooltip>Something</Tooltip>
+</TooltipTrigger>
+```
+
+### Offset and cross offset
+
+The Tooltip's offset with respect to its trigger can be adjusted using the `offset` and
+`crossOffset` props. The `offset` prop controls the spacing applied along the main axis between the element and its
+trigger whereas the `crossOffset` prop handles the spacing applied along the cross axis.
+
+Below is a tooltip offset by an additional 50px above the trigger.
+```tsx example
+<TooltipTrigger offset={50}>
+  <ActionButton aria-label="Offset from trigger">Offset</ActionButton>
+  <Tooltip>This will shift up.</Tooltip>
+</TooltipTrigger>
+```
+Below is a tooltip cross offset by an additional 100px to the right of the trigger.
+```tsx example
+<TooltipTrigger crossOffset={100} placement="bottom">
+  <ActionButton aria-label="Cross Offset from trigger">Cross Offset</ActionButton>
+  <Tooltip>This will shift over to the right.</Tooltip>
+</TooltipTrigger>
+```
+
+## Events
+
+TooltipTriggers accepts an `onOpenChange` handler which is triggered whenever the Tooltip is shown or hidden.
+
+The example below uses `onOpenChange` to update a separate element with the current open state of the
+Dialog.
+
+```tsx example
+function Example() {
+  let [state, setState] = React.useState(false);
+
+  return (
+    <Flex alignItems="center" gap="size-100">
+      <TooltipTrigger onOpenChange={(isOpen) => setState(isOpen)}>
+        <ActionButton aria-label="Resize"><Resize /></ActionButton>
+        <Tooltip>Text auto-resized in symbol instance.</Tooltip>
```

Comment:
this tooltip content is a little confusing as an example. Maybe just "Resize text" ?

Reply:
haha, you can blame spectrum for this sentence https://spectrum.corp.adobe.com/page/tooltip/#Maximum-width